config/zephyr/Kconfig

```diff
@@ -108,6 +108,17 @@ config CHIP_OTA_REQUESTOR
 	  Device Firmware Upgrade by quering and downloading a new firmware image
 	  from an external OTA Provider node.
 
+config CHIP_OPERATIONAL_TIME_SAVE_INTERVAL
+	int "Interval of saving node operation time to flash in hours unit"
+	default 10
+	range 1 24
+	help
+	  Interval in hours with which the node operation time is saved to the flash
+	  memory. Selected value is a trade of between performing often saves to know
+	  precisely operation time in case of device reboot and maximizing flash memory
+	  lifetime. Default value of 10 hours allows for over 11 years of flash memory operation,
```

Comment:
This estimate is only accurate if you assume it will be written into the same block all the time. Typically, flash drivers try to uniformly wear the flash, so it's probably bigger than that.

Reply:
Yeah I know, but isn't it reasonable to assume the worst case?

Reply:
I can also just do not provide such information at all

Reply:
Yeah, I'm not sure, but maybe it's better not to provide this information to not scare users needlessly in case somebody would like to save it more frequently. :)